src/lamp_py/performance_manager/alerts.py

```diff
@@ -48,36 +50,18 @@ class AlertsS3Info:
             ("service_effect_text.translation.text", pyarrow.string()),
             ("timeframe_text.translation.text", pyarrow.string()),
             ("recurrence_text.translation.text", pyarrow.string()),
-            (
-                "created_datetime",
-                pyarrow.timestamp("ms", tz="America/New_York"),
-            ),
-            ("created_timestamp", pyarrow.uint64()),
-            (
-                "last_modified_datetime",
-                pyarrow.timestamp("ms", tz="America/New_York"),
-            ),
-            ("last_modified_timestamp", pyarrow.uint64()),
-            (
-                "last_push_notification_datetime",
-                pyarrow.timestamp("ms", tz="America/New_York"),
-            ),
-            ("last_push_notification_timestamp", pyarrow.uint64()),
-            (
-                "closed_datetime",
-                pyarrow.timestamp("ms", tz="America/New_York"),
-            ),
-            ("closed_timestamp", pyarrow.uint64()),
-            (
-                "active_period.start_datetime",
-                pyarrow.timestamp("ms", tz="America/New_York"),
-            ),
-            ("active_period.start_timestamp", pyarrow.uint64()),
-            (
-                "active_period.end_datetime",
-                pyarrow.timestamp("ms", tz="America/New_York"),
-            ),
-            ("active_period.end_timestamp", pyarrow.uint64()),
```

Comment:
`uint64` -> `int64`

remove tz info from timestamp column types. they are localized to boston, but without the tz info. downstream users can assume they are in boston time.